### ISSUE TITLE
SOSTENIAMO LA CULTURA! APPELLO AL GOVERNO E ALLE REGIONI

### ISSUE THREAD

**Issue by ehack-italy** [emergenzeHack/covid19italia_segnalazioni#1007]
<pre><yamldata>
Da_chi_offerta: Assessorati alla Cultura in Italia
Descrizione: 'La sospensione totale delle attività culturali è stata la prima delle
  misure di riduzione della socialità, prima nelle regioni settentrionali e poi in
  tutta Italia. Ora sono indispensabili e non rinviabili misure che ne assorbano gli
  impatti.

  La petizione completa per chiedere misure a sostegno della cultura al seguente link

  https://www.ipetitions.com/petition/coronavirus-sosteniamo-la-cultura-appello-al-2'
Destinatari: tutti
Link: https://www.ipetitions.com/petition/coronavirus-sosteniamo-la-cultura-appello-al-2
Natura: culturale-ricr
Titolo: SOSTENIAMO LA CULTURA! APPELLO AL GOVERNO E ALLE REGIONI
</yamldata></pre>

**Comment by chiccap** [comment]
@cristigalas abbiamo deciso di non pubblicare petizioni vero?

**Comment by cristigalas** [comment]
@chiccap no, no, le pubblichiamo ma non quelle lanciate senza riferimenti chiari su chi le lancia, contatti, ecc. se sono su piattaforme ad hoc, come questa, le accettiamo.

**Comment by cristigalas** [comment]
Qui i promotori sono vari assessori alla cultura... premiamoli nel titolo.
